lib/models/seed_model.dart

```diff
@@ -0,0 +1,19 @@
+class SeedModel {
+  final String seed;
+  final String description;
+  final String expirationDate;
+
+  SeedModel({
```

Comment:
**suggestion**: make sure to always use `const` whenver possible! You've made a good job, you have forgotten adding it here 😄 

I suggest you to use more linter rules to avoid forgetting this kind of things

Reply:
Hello @alberto-sf, it's a pleasure to meet you!

Thank you for these suggestions, it's much appreciated. I will certainly apply this feedback to my work going forward 😄